### ISSUE TITLE
Unable to load powershell

### ISSUE THREAD

**Issue by mgpinf** [PowerShell/PSReadLine#2961]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

### Screenshot

![image](https://user-images.githubusercontent.com/56510798/140850780-1f67dd19-a7d1-4c06-ae84-f60e23681023.png)
![image](https://user-images.githubusercontent.com/56510798/140850811-1328c607-2d7e-4e00-85c8-0319c64d2a3c.png)

### Environment data

```console
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 156
BufferHeight: 42
```

### Steps to reproduce

Open powershell

### Expected behavior

Smooth launch of powershell

### Actual behavior

![image](https://user-images.githubusercontent.com/56510798/140850517-51b67712-8b9f-477e-8e22-854ca3c5b527.png)

**Comment by skittles9823** [comment]
Opening windows terminal as administrator and updating PSReadLine manually to 2.2.0-beta4 fixed this for me. 

`pwsh.exe -noprofile -command "Install-Module PSReadLine -Force -SkipPublisherCheck -AllowPrerelease"`

**Comment by daxian-dbw** [comment]
> Environment
> PSReadLine: 2.2.0-beta2

This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
